Source/ChocolateyGui/App.xaml.cs

```diff
@@ -172,96 +174,100 @@ private static void ParseArgumentsAndRunCommand(string[] args)
+            var configService = new ConfigService(database);
+            var configuration = configService.GetSettings();
 
             switch (featureCommand)
             {
                 case FeatureCommandType.list:
-                    ListFeatures(configService, regularOutput);
+                    ListFeatures(configuration, regularOutput);
                     break;
                 case FeatureCommandType.disable:
-                    DisableFeature(configService, featureName);
+                    DisableFeature(configService, configuration, featureName);
                     break;
                 case FeatureCommandType.enable:
-                    EnableFeature(configService, featureName);
+                    EnableFeature(configService, configuration, featureName);
                     break;
             }
         }
 
-        private static void ListFeatures(IConfigService configService, bool regularOutput)
+        private static void ListFeatures(AppConfiguration configuration, bool regularOutput)
         {
-            var settings = configService.GetSettings();
-
             var properties = typeof(AppConfiguration).GetProperties();
             foreach (var property in properties)
             {
-                var propertyName = property.Name;
+                var propertyName = property.Name;
 
                 if (property.Name != "Id")
                 {
                     var attributes = property.GetCustomAttributes(typeof(LocalizedDescriptionAttribute), true);
                     var attribute = attributes.Length > 0 ? (LocalizedDescriptionAttribute)attributes[0] : null;
-                    var descriptionText = attribute?.Description;
+                    var descriptionText = attribute?.Description;
+
+                    var propertyValue = (bool)property.GetValue(configuration);
 
-                    var propertyValue = (bool)property.GetValue(settings);
-
                     if (regularOutput)
                     {
                         Console.WriteLine("{0} {1} - {2}".format_with(propertyValue ? "[x]" : "[ ]", propertyName, descriptionText));
                     }
                     else
-                    {
+                    {
                         Console.WriteLine("{0}|{1}|{2}".format_with(propertyName, !propertyValue ? "Disabled" : "Enabled", descriptionText));
                     }
                 }
-            }
+            }
         }
 
-        private static void DisableFeature(IConfigService configService, string featureName)
+        private static PropertyInfo GetFeatureProperty(string featureName)
         {
             var featureProperty = typeof(AppConfiguration).GetProperties().FirstOrDefault(f => f.Name.ToLowerInvariant() == featureName.ToLowerInvariant());
+
             if (featureProperty == null)
             {
                 Console.WriteLine("Feature '{0}' not found", featureName);
-                return;
+                Environment.Exit(-1);
```

Comment:
Fail fast :)?

Reply:
@RichiCoder1 ha ha, yeah, something like that 😄 Decided that since it was running from the command line, returning with a hard exit, made sense.